x-pack/plugins/stack_connectors/server/connector_types/openai/openai.ts

```diff
@@ -259,5 +260,24 @@ const transformToString = () => {
       const newChunk = encoder.encode(nextChunk);
       callback(null, newChunk);
     },
+    flush(callback) {
+      // Emit an additional chunk with the content of lineBuffer if it has length
+      if (lineBuffer.length > 0) {
+        const nextChunk = [lineBuffer]
+          .map((str) => str.substring(6))
+          .filter((str) => !!str && str !== '[DONE]')
+          .map((line) => {
+            const openaiResponse = JSON.parse(line);
+            return openaiResponse.choices[0]?.delta.content ?? '';
+          })
+          .join('');
+        const newChunk = encoder.encode(nextChunk);
+        callback(null, newChunk);
```

Comment:
@dgieselaar I hoped this callback would put the buffer through the transform function again, but it just pushes directly to the stream. I'm worried this is wrong